src/node/chainstatemanager_args.cpp

```diff
@@ -60,8 +60,7 @@ util::Result<void> ApplyArgsManOptions(const ArgsManager& args, ChainstateManage
         script_threads += GetNumCores();
```

Comment:
Slightly unrelated, but this doesn't actually return the number of cores, but [number of concurrent threads](https://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency)